src/pages/Transaction/TransactionComp/TransactionComp.tsx

```diff
@@ -53,36 +53,22 @@ export const TransactionComp = ({
   const outputsPage = page_of_outputs && Number.isNaN(+page_of_outputs) ? 1 : +page_of_outputs
   const pageSize = Number.isNaN(+page_size) ? PAGE_SIZE : +page_size
 
-  const { data: displayInputs, isFetching: isInputsLoading } = useQuery(
+  const { data: _displayInputs, isFetching: isInputsLoading } = useQuery(
     ['transaction_inputs', txHash, inputsPage, pageSize],
-    async () => {
-      try {
-        const res = await explorerService.api.fetchCellsByTxHash(txHash, 'inputs', { no: inputsPage, size: pageSize })
-        return res
-      } catch (e) {
-        return emptyList
-      }
-    },
-    {
-      initialData: emptyList,
-    },
+    () => explorerService.api.fetchCellsByTxHash(txHash, 'inputs', { no: inputsPage, size: pageSize }),
   )
 
-  const { data: displayOutputs, isFetching: isOutputsLoading } = useQuery(
+  const { data: _displayOutputs, isFetching: isOutputsLoading } = useQuery(
     ['transaction_outputs', txHash, outputsPage, pageSize],
-    async () => {
-      try {
-        const res = await explorerService.api.fetchCellsByTxHash(txHash, 'outputs', { no: outputsPage, size: pageSize })
-        return res
-      } catch (e) {
-        return emptyList
-      }
-    },
-    {
-      initialData: emptyList,
-    },
+    () =>
+      explorerService.api
+        .fetchCellsByTxHash(txHash, 'outputs', { no: outputsPage, size: pageSize })
+        .catch(() => emptyList),
   )
 
+  const displayInputs = _displayInputs ?? emptyList
+  const displayOutputs = _displayOutputs ?? emptyList
```

Comment:
`emptyList` can be set as default in `{ data: displayOutputs }` to avoid this nullish coalescing

Reply:
Because the tx data is immutable, the query here uses the `not-auto-refresh`, and the `react-query` set `default` conflicts with the `not-auto-refresh`, resulting in the `default` value being used all the time.

Reply:
The default value set by `const { data: displayOuutputs = emptyList }` is not handled by `react-query`, it's just a sugar syntax for 

```ts
const { data: _displayOutputs } 
const displayOutputs = _displayOutputs ?? emptyList
```

Reply:
> The default value set by `const { data: displayOuutputs = emptyList }` is not handled by `react-query`, it's just a sugar syntax for
> 
> ```ts
> const { data: _displayOutputs } 
> const displayOutputs = _displayOutputs ?? emptyList
> ```

got it